code/modules/ruins/spaceruin_code/oldstation.dm

```diff
@@ -118,7 +118,7 @@
 	var/list/overlays = ..()
 	if(machine_stat & (NOPOWER|BROKEN))
 		return overlays
-	overlays += busy ? "red" : "green"
+	overlays += busy || !mod_unit ? "red" : "green"
```

Comment:
is this intended to be `busy || (!mod_unit ? "red" : "green")`, or `(busy || !mod_unit) ? "red" : "green"`? this scares me

Reply:
put this in parentheses anyway to make this more obvious

```suggestion
	overlays += (busy || !mod_unit) ? "red" : "green"
```